local/php/php_server.go

```diff
@@ -224,8 +233,14 @@ func (p *Server) serveFastCGI(env map[string]string, w http.ResponseWriter, r *h
 	max := 10
 	i := 0
 	for {
-		if fcgi, err = fcgiclient.Dial("tcp", p.addr); err == nil {
-			break
+		if p.Version.IsFPMServer() {
+			if fcgi, err = fcgiclient.Dial("unix", p.addr); err == nil {
+				break
+			}
+		} else {
+			if fcgi, err = fcgiclient.Dial("tcp", p.addr); err == nil {
+				break
+			}
```

Comment:
I'm wondering if we should not refactor this to have a simpler dialer internal function here.
wdyt @fabpot ?